.drone.yml

```diff
@@ -386,7 +386,23 @@ services:
         path: /tmp
 
 steps:
-  - name: gce
+  - name: osctl-cluster-create
+    image: autonomy/build-container:latest
```

Comment:
maybe later see if we can switch to autonomy/installer and skip osctl-linux build?